src/components/map/page.module.css

```diff
@@ -0,0 +1,33 @@
+.mapButton {
+    position: absolute;
+    right: 10px;
+    top: 76%;
+    transform: translateY(-50%);
+    z-index: 1000;
+    background: white;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    padding: 0;
+    cursor: pointer;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.3);
+}
+
+.mapButton img {
+    width: 40px;
+    height: 40px;
+}
+
+@media (min-width: 430px) and (max-width: 500px) {
+    .mapButton {
+        width: 40px;
+        height: 50px;
+    }
+}
+
+@media (max-width: 376px) {
+    .mapButton {
+        width: 40px;
+        height: 40px;
+        top: 73%;
+    }
```

Comment:
change `max-width` to 429px so it can consistently transition between the two media queries instead of having a gap between the two screen widths